### PR TITLE
Document canonical link was empty

### PR DESCRIPTION
**Pull request by userotti**
Se mar as ek bietjie moet edit.

### CODE REVIEW

File: content/seo/document-canonical-link-was-empty.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+The canonical link element on the page needs to inform search engines about the appropriate page to use to define the content of the current page. This is important when the current page might contain content very similar to other pages on your website. Search engines tend to penalize duplicate content.
```
Comment (Johanndutoit) [inline content/seo/document-canonical-link-was-empty.md]: Like this, think we just need another paragraph that describe this error itself. So why you would not want to have a canonical link with a empty href

File: content/seo/page-is-blocked-for-any-search-indexing-by-googlebot-feature-phone.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+Klein editb fdgdf
```
Comment (Johanndutoit) [inline content/seo/page-is-blocked-for-any-search-indexing-by-googlebot-feature-phone.md]: Think this was a oopsie from our testing on Friday :)

**Comment by Johanndutoit** [comment]
Look good see comments, also just remove the other file from the PR so we keep them all on track.

**Comment by userotti** [comment]
Ah yes, ek gan bietjie kyk johann.

**Comment by Johanndutoit** [comment]
Noice, looks good to me

**Pull request merged**